app/screens/thread/thread_base.js

```diff
@@ -14,21 +15,29 @@ import DeletedPost from 'app/components/deleted_post';
 import {popTopScreen, mergeNavigationOptions} from 'app/actions/navigation';
 import {TYPING_HEIGHT, TYPING_VISIBLE} from '@constants/post_draft';
 
+import ThreadFollow from './thread_follow';
+
+Navigation.registerComponent('ThreadFollow', () => ThreadFollow);
```

Comment:
why is this being registered here and not in screens/index ? I'd like to keep the registration of Navigation components in one place